sdk/spring/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/exception/CosmosExceptionUtils.java

```diff
@@ -28,7 +28,7 @@ public CosmosExceptionUtils() {
      * @param responseDiagnosticsProcessor response diagnostics processor
      * @param <T> type class of Mono
      * @return Mono instance
-     * @throws CosmosAccessException for operations on cosmos db
```

Comment:
When would it not be a ComsosAccessException? Technically this documentation change is indicating a breakign change - I thought all the new exceptions extend CosmsoAccessException - so, the comment shouldn't be necessary?

Reply:
Apologies this was leftover from the old implementation, fixed